CalendarFXView/src/main/java/com/calendarfx/model/EntryEditParameter.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (C) 2017 Dirk Lemmermann Software & Consulting (dlsc.com)
+ * <p>
+ * This file is part of CalendarFX.
+ */
+package com.calendarfx.model;
```

Comment:
This class also does not belong into the model package. Please place it inside DateControl, next to the policy callback that requires it.